debian/source/format

```diff
@@ -1 +1 @@
-3.0 (quilt)
+3.0 (native)
```

Comment:
I'm not familiar with these formats and had copied from a different debian package. what's the difference?

Reply:
I did what Thomas told me :)  I'm not familiar either.